packages/now-cli/src/commands/teams/list.js

```diff
@@ -78,4 +94,14 @@ export default async function({ teams, config, apiUrl, token }) {
     teamList.map(team => [team.current, team.value, team.name]),
     [1, 5]
   );
+
+  if (pagination && pagination.count === 20) {
+    const prefixedArgs = getPrefixedFlags(argv);
+    const flags = getCommandFlags(prefixedArgs, ['_', '--next', '-N', '-d']);
+    const nextCmd = `${getPkgName()} teams ls${flags} --next ${
+      pagination.next
+    }`;
+    console.log(); // empty line
```

Comment:
Just keeping it consistent.

```suggestion
    output.log(); // empty line
```

Reply:
@TooTallNate That won't work because `output.log()` will print `>` and we don't want that.